heu/pybind/phe_test.py

```diff
@@ -133,27 +133,5 @@ def test_batch_encoding(self):
         # BatchEncoding 的减法慎用，只有当每个元素都是正整数时才可以使用
         self.assertEqual(bc.decode(self.decryptor.decrypt(self.evaluator.sub(ct1, ct2))), (123 - 789, 456 - 101112))
 
-
-class ServerClientCase(unittest.TestCase):
```

Comment:
why remove this UT ?

Reply:
This UT was using an invalid schema type, now updated with a proper one...